docker/dev/django/Dockerfile

```diff
@@ -15,19 +15,17 @@ RUN useradd evalai -G webapps
 RUN mkdir -p /var/log/evalai/ && chown -R evalai /var/log/evalai/ && chmod -R u+rX /var/log/evalai/
 RUN mkdir -p /var/run/evalai/ && chown -R evalai /var/run/evalai/ && chmod -R u+rX /var/run/evalai/
 
-RUN apt-get install -y supervisor
-RUN mkdir -p /var/log/supervisor
-
-ADD ./docker/django/evalai_supervisor.conf /etc/supervisor/conf.d/evalai.conf
-
 WORKDIR /code
 
 ADD ./ /code/
 
-RUN touch settings/dev.py
-COPY settings/dev.sample.py /code/settings/dev.py
-
 RUN pip install -r requirements/dev.txt
 
-CMD ["sh", "/code/docker/django/container-start.sh"]
+ENV POSTGRES_NAME="postgres"
```

Comment:
I think you can put these environment variables in `dev.env` file. For more reference, please see https://docs.docker.com/compose/env-file/